canopeum_frontend/src/locale/en/social.json

```diff
@@ -29,7 +29,7 @@
   },
   "widgets": {
     "create": "add a new widget",
-    "save": "save",
+    "submit": "submit",
     "cancel": "cancel",
```

Comment:
It's arguable whether these two should even exist in favor of the generic translation. Can be removed separatly